src/ResourceManager/RecoveryServices/Commands.RecoveryServices/ChangeLog.md

```diff
@@ -0,0 +1,35 @@
+<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+* Moved client from Hydra to Swagger
```

Comment:
Hydra/Swagger don't mean anything to customers, please describe the actual interface changes to cmdlets (new cmdlets, new parameters, new output, shouldprcess support)